### DISCUSSION TITLE
Does Persona have function that custom tooltip?

### DISCUSSION THREAD

**Discussion by giabaoye** [microsoft/fluentui#31148] [Q&A]
I have a problem that when I display the tooltip, it position not correctly right, it looks like this
<img width="205" alt="IMG_5673" src="https://github.com/microsoft/fluentui/assets/21209285/d417e59d-a184-4c94-aea0-bebca2aa5b6a">
 Can I custom it to be normal again?

**Comment by pradeept95** [comment]
Hi @baokun2115 , could you please share the code how you're using currently?
